config/config_test.go

```diff
@@ -38,19 +38,19 @@ func TestParse(t *testing.T) {
 	}{
 		{
 			in:  in{config: []byte(`{"ignitionVersion": 1}`)},
-			out: out{config: types.Config{Ignition: types.Ignition{Version: types.IgnitionVersion(v2_0.MaxVersion)}}},
+			out: out{config: types.Config{Ignition: types.IgnitionVersion{Version: v2_0.MaxVersion.String()}}},
 		},
 		{
 			in:  in{config: []byte(`{"ignition": {"version": "1.0.0"}}`)},
 			out: out{err: v1.ErrVersion},
 		},
 		{
 			in:  in{config: []byte(`{"ignition": {"version": "2.0.0"}}`)},
-			out: out{config: types.Config{Ignition: types.Ignition{Version: types.IgnitionVersion(types.MaxVersion)}}},
+			out: out{config: types.Config{Ignition: types.IgnitionVersion{Version: types.MaxVersion.String()}}},
 		},
 		{
 			in:  in{config: []byte(`{"ignition": {"version": "2.1.0-experimental"}}`)},
```

Comment:
Added a test, but it's less pretty than the other tests (it's string comparison) since [go-semver doesn't export the error it gives for this](https://github.com/coreos/go-semver/blob/master/semver/semver.go#L76).